test/exportable/ExportableInstance.test.ts

```diff
@@ -1,3 +1,4 @@
+import { FshCode } from 'fsh-sushi/dist/fshtypes';
```

Comment:
I think the usual way we import this is `import { fshtypes } from 'fsh-sushi` and then access `FshCode` by doing `fshTypes.FshCode` or pulling `FshCode` off of `fshtypes` separately like `const  const { FshCode } = fshtypes;`.

Reply:
Ahhh yes, this was done as an auto import. The most recent commit updates this test to the usual motif that we use.